h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -116,24 +120,40 @@ public void initMetalearnerParams(Metalearner.Algorithm algo) {
   protected Frame predictScoreImpl(Frame fr, Frame adaptFrm, String destination_key, Job j, boolean computeMetrics, CFuncRef customMetricFunc) {
     Frame levelOneFrame = new Frame(Key.<Frame>make("preds_levelone_" + this._key.toString() + fr._key));
 
-    // TODO: we should be able to parallelize scoring of base models
+    List<H2O.H2OCountedCompleter> tasks = new ArrayList<>();
```

Comment:
this is emulating what `Futures` are meant to do, I would rather use Futures or switch to MRLocal (see my other comment)

Reply:
good point, those `H2OCounterCompleter` are `Future`s